config/security/500-webhook.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook
+  namespace: knative-security
+  labels:
+    security.knative.dev/release: devel
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: knative-security
+      role: webhook
+  template:
+    metadata:
+      labels:
+        app: knative-security
+        role: webhook
+        security.knative.dev/release: devel
```

Comment:
This is not present on the webhook Deployment's Pods. Should it be?

Reply:
I copied it over from the other webhook yaml. I think the idea is to have `*/release: devel` label on every resource we create?

Reply:
That makes sense to me. What I meant to point out is that the Webhook and Controller's Pods are inconsistent. The Webhook Pods have this label. The Controller Pods do not. I would prefer we either have the label in both or neither.